library_generation/generate_library.sh

```diff
@@ -54,6 +54,7 @@ shift # past argument or value
 done
 
 script_dir=$(dirname "$(readlink -f "$0")")
+output_folder="$(pwd)/output"
```

Comment:
Can we add this folder to `.gitignore`? For both running from root folder and showcase.

Reply:
added. I included `sdk-platform-java/output` `sdk-platform-java/library_generation/output` and `sdk-platform-java/showcase/scripts/output`